tests/Application/templates/bundles/SyliusAdminBundle/Order/Show/_payment.html.twig

```diff
@@ -17,10 +17,28 @@
     {% if sm_can(payment, 'complete', 'sylius_payment') %}
         <div class="ui segment">
             <form action="{{ path('sylius_admin_order_payment_complete', {'orderId': order.id, 'id': payment.id}) }}" method="post" novalidate>
-                <input type="hidden" name="_csrf_token" value="{{ csrf_token(payment.id) }}" />
                 <input type="hidden" name="_method" value="PUT">
+                <input type="hidden" name="_csrf_token" value="{{ csrf_token(payment.id) }}" />
                 <button type="submit" class="ui icon labeled tiny blue fluid loadable button"><i class="check icon"></i> {{ 'sylius.ui.complete'|trans }}</button>
             </form>
         </div>
+        {% if payment.details['payment-link'] is defined %}
```

Comment:
Why this change? Shouldn't we add only tests?

Reply:
It's part of installation AdminOrderCreation plugin, same as added Repositories